_data/apis.yml

```diff
@@ -111,4 +111,14 @@
   tags: [data, turismo]
   language: [it, en]
   slug: opendata-ert
+- openapi: https://idrogeo.isprambiente.it/api
+  title: Idrogeo API 
+  abstract: >-
+    Consulta mappe, dati e indicatori sul dissesto idrogeologico in Italia.
+  contact:
+    name: ISPRA
+    logo: https://www.isprambiente.gov.it/it/archivio/notizie-e-novita-normative/notizie-ispra/2017/05/nota-ministero-dellambiente/@@images/cc88e735-4212-44de-818c-dde31a6597a9.jpeg
```

Comment:
We should avoid hotlinking the logo

Reply:
api specs are hotlinked too ...

Reply:
Those are expected to always be there at the same URL and (hopefully) be updated by the API owners. The logo will probably vanish next time the website is retooled.